app/scripts/modules/core/src/managed/DeployingIntoManagedClusterWarning.tsx

```diff
@@ -45,9 +45,9 @@ export const DeployingIntoManagedClusterWarning = ({ app, formik }: IDeployingIn
   return (
     <div className="alert alert-danger">
       <p>
-        <b>🌈 Spinnaker is continuously managing this resource.</b>
+        <b>🌈 Spinnaker is managing this resource.</b>
       </p>
-      <p>Any changes you make to this cluster will be stomped in favor of the declarative configuration.</p>
+      <p>Any changes you make to this cluster will be overridden in favor of the desired state.</p>
```

Comment:
While I think "overriden" is better, I will miss "stomped".

Reply:
same — may it rest in peace. 🕯️